src/main/java/calculationEngine/environment/CeItemEffects.java

```diff
@@ -0,0 +1,74 @@
+package calculationEngine.environment;
+
+import calculationEngine.entities.CeStats;
+import config.BattleConstants;
+import config.LootConfig;
+import org.json.JSONObject;
+
+public class CeItemEffects {
+
+    private effects effect;
+    private String description;
+    private int value;
+
+    private static final boolean debug = BattleConstants.battleDebug;
+
+    private enum effects{
+        healing,
+        increaseSpeed,
+        increaseAttack,
+        increaseCritBonus,
+        increaseDefense,
+        catchBeast,
+        defaultEffect
+    }
+
+    public CeItemEffects(JSONObject effectObject){
+        this.effect = effects.valueOf(effectObject.getString("type"));
+        this.value = effectObject.getInt("value");
+        this.description = effectObject.getString("description");
+    }
+
+    public void use(CeStats ceStats){
+        switch (this.effect.name()){
+            case "healing":
+                int hitPoints = ceStats.getCurrentHitPoints();
+                hitPoints += this.value;
+                ceStats.setCurrentHitPoints(hitPoints);
+                if (debug) System.out.println("[CeItemEffects] healing by " + this.value + " points");
```

Comment:
you could create a method, "addToStat" in which you say to which stat you want to add (perhaps with ENUM) and how much you want to add.

Reply:
I don't think it's neccessary, because it doens't change the problem. For every stat that has to change, you have to call different methods. So if i add another Method this will result in a switch case which has another switch case in it. So you just duplicate the whole thing and you have it twice. IMO it makes it even worse.